exec/java-exec/src/main/java/org/apache/drill/exec/work/metadata/ServerMetaProvider.java

```diff
@@ -76,7 +76,7 @@
       .setReadOnly(false)
       .setGroupBySupport(GroupBySupport.GB_UNRELATED)
       .setLikeEscapeClauseSupported(true)
-      .setNullCollation(NullCollation.NC_AT_END)
+      .setNullCollation(NullCollation.NC_HIGH)
```

Comment:
@jinfengni  - See @vdiravka's comments in the Jira.  The Drill documentation at https://drill.apache.org/docs/order-by-clause/#usage-notes says NULLs sort highest.  If the doc is wrong, then we should fix the doc.